spec/queries/patient_states/disaggregated_patient_count_query_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+describe PatientStates::DisaggregatedPatientCountQuery do
+  around do |example|
+    with_reporting_time_zone { example.run }
+  end
+
+  let(:query) {
+    Reports::PatientState.all
+  }
+
+  describe ".disaggregate_by_gender" do
+    it "returns the query disaggregated by gender" do
+      _facility_1_patient_1 = create(:patient, gender: :male)
```

Comment:
This test would fail if all these genders are not supported in the country. 
```suggestion
      Patient::GENDERS. each  { |gender| create(:patient, gender: gender) }
```